CHANGELOG.md

```diff
@@ -22,17 +22,21 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
 - Scroll to top functionality on footer
 - Added `/modules/util/web-storage-proxy.js` utility module.
 - Added `/modules/util/js-loader.js` utility module.
+- Adds ESLint check for `@todo` jsdoc syntax.
```

Comment:
In [GHE]/github-enterprise-assets/0000/0370/0000/9856/8d11c1fc-ecd4-11e4-8319-152117954fdd.png we're only tagging at beta. Are you suggesting creating placeholders for tags we will actually tag later on at beta? Or something else? I'd favor keeping everything in the un-released section till the actual time of tagging. It means it gets looked at many times before being tagged.